source/maven-archetypes/index.md

```diff
@@ -3,155 +3,146 @@ layout: default
 title: Maven Archetypes (WIP)
 ---
 
-#Struts 2 Maven Archetypes# {#PAGE_108820}
+# Struts 2 Maven Archetypes
 
-Struts 2 provides several [Maven](http://maven\.apache\.org/)^[http://maven\.apache\.org/] archetypes that create a starting point for our own applications\.
+Struts 2 provides several [Maven](http://maven.apache.org/) archetypes that create a starting point for our own applications.
 
 **Contents**
+{:.no_toc}
 
-###Quickstart###
+* Will be replaced with the ToC, excluding a header
+{:toc}
 
-A recommended way to start with Struts2 archetypes is to work with the archetype catalog\.
+### Quickstart
```

Comment:
It would be better to use `##` instead of `###`

Reply:
Oh... Thank you very much for that 👍 . I also fixed others :)